src/web/resources/report.js

```diff
@@ -35,6 +35,94 @@ function Element(tagname, props, children) {
     return $elt;
 }
 
+var renames = {};
+renames["imp-start"] = "Improved start"
+renames["apx-start"] = "Approximate start"
+renames["uni-start"] = "Regressed from start"
+renames["ex-start"]  = "Exact start"
+renames["eq-start"]  = "Equal start"
+renames["lt-start"]  = "Less than start"
+renames["gt-start"]  = "Greater than start"
+renames["gt-target"] = "Greater than target"
+renames["eq-target"] = "Equal than target"
+renames["lt-target"] = "Less than target"
+renames["error"]     = "Error"
+renames["timeout"]   = "Timeout"
+renames["crash"]     = "Crash"
```

Comment:
My man, why not just define an object with the fields already in there?

    var renames = { "imp-start": "Improved start", ... }